arm-redis/2015-08-01/swagger/redis.json

```diff
@@ -595,6 +646,32 @@
         "keyType"
       ],
       "description": "Specifies which redis access keys to reset."
+    },
+    "RedisRebootParameters": {
+      "properties": {
+        "rebootType": {
+          "type": "string",
+          "description": "Which redis node(s) to reboot. Depending on this value data loss is possible.",
+          "enum": [
+            "PrimaryNode",
+            "SecondaryNode",
+            "AllNodes"
+          ],
+          "x-ms-enum": {
+            "name": "RebootType",
+            "modelAsString": false
+          }
+        },
+        "shardId": {
+          "type": "integer",
+          "format": "int32",
+          "description": "In case of cluster cache, this specifies shard id which should be rebooted."
```

Comment:
Ah I see, In that case, please update the description as follows:

> "In case of cluster cache, this specifies **the** shard id ~~which~~ **that** should be rebooted."